frontend/model/notifications/mainNotificationsMixin.js

```diff
@@ -262,16 +106,7 @@ const periodicNotificationEntries = [
 
```

Comment:
(Unless it was intentional) Is there a reason that this notification entry was not moved to `mainPeriodicNotificationEntries.js` along with others?

Reply:
It was intentional. Since we added it to fix an issue that we don't fully understand and seemed to affect the UI, I left it there to run in the UI.